### ISSUE TITLE
Missing Latest docker-stable 

### ISSUE THREAD

**Issue by rjacquemin** [jitsi/docker-jitsi-meet#1124]
Hi 

Since latest release was announced - 2021-09-10
https://github.com/jitsi/jitsi-meet-release-notes/blob/master/CHANGELOG-WEB.md 

I was looking for latest stable version of the docker-jitsi-meet 

I was expecting to find 
https://github.com/jitsi/docker-jitsi-meet/releases/tag/stable-6293  

latest version available  is : 
https://github.com/jitsi/docker-jitsi-meet/releases/tag/stable-6173

Thanks in advance

**Comment by saghul** [comment]
Yes we'll make a new release soon. It wasn't stable enough and a new one is coming.
